vuepress/docs/docs/README.md

```diff
@@ -21,6 +21,7 @@ To use `caliban`, add the following dependency to your `build.sbt` file:
 The following modules are optional:
 
 ```scala
+"com.github.ghostdogpr" %% "caliban-quick"      % "2.4.2" // quick adapter
```

Comment:
Should we add an example of running the quick server right on the Getting Started page? Sounds like it would be good.

Reply:
I had to re-arrange a few things to make it coherent. Not sure if what I went with is the best approach - let me know

Reply:
Looks good!